src/lang_utils/error_codes/M0195.md

```diff
@@ -0,0 +1,34 @@
+# M0195
+
+This warning means that you called a function that demands elevated (`system`) privileges,
+without manifestly passing the privilege.
+
+Dubious code example:
+
+```motoko
+Cycles.add(2_000_000_000_000);
+```
+
+If you encounter this warning, you should be giving some thought if the call can cause adverse behaviour,
+and if you agree to perform the call you can add a manifest instantiation `<system>` (possibly followed
+by further type parameters) to suppress the warning:
+
+```motoko
+Cycles.add<system>(2_000_000_000_000);
+```
+
+## Background
+
+Some builtin functions are defined to demand elevated privileges, as they perform system activities related
+to resource management, involving values or privilege elevation. E.g. the builtin function
+
+```motoko
+setTimer : <system>(Nat64, Bool, () -> async ()) -> Nat
+```
+
+installs a callback which eventually gets invoked in an `async` context and as such can send messages and
+cycles. In order to avoid calls to `setTimer` from unprivileged synchronous code, its signature declares a demand
```

Comment:
```suggestion
cycles. In order to avoid calls to `setTimer` from unprivileged synchronous code, its signature requires system capability, represented by the pseudo type parameter 'system';
```